builder/osc/common/run_config_test.go

```diff
@@ -13,10 +13,10 @@ import (
 func init() {
 	// Clear out the OUTSCALE access key env vars so they don't
 	// affect our tests.
-	os.Setenv("OUTSCALE_ACCESS_KEY_ID", "")
-	os.Setenv("OUTSCALE_ACCESS_KEY", "")
-	os.Setenv("OUTSCALE_SECRET_ACCESS_KEY", "")
-	os.Setenv("OUTSCALE_SECRET_KEY", "")
+	os.Setenv("OUTSCALE_ACCESS_KEY_ID", "") || os.Setenv("OSC_ACCESS_KEY_ID", "")
+	os.Setenv("OUTSCALE_ACCESS_KEY", "") || os.Setenv("OSC_ACCESS_KEY", "")
+	os.Setenv("OUTSCALE_SECRET_ACCESS_KEY", "") || os.Setenv("OSC_SECRET_KEY", "")
```

Comment:
Not sure if this a copy paster too, just in case
```suggestion
	os.Setenv("OUTSCALE_SECRET_ACCESS_KEY", "") || os.Setenv("OSC_SECRET_ACCESS_KEY", "")
```